src/sentry/rules/processor.py

```diff
@@ -37,14 +37,15 @@ def message(self):
 class RuleProcessor(object):
     logger = logging.getLogger('sentry.rules')
 
-    def __init__(self, event, is_new, is_regression, is_new_group_environment):
+    def __init__(self, event, is_new, is_regression, is_new_group_environment, is_reappeared):
         self.event = EventCompatibilityProxy(event)
         self.group = event.group
         self.project = event.project
 
         self.is_new = is_new
         self.is_regression = is_regression
         self.is_new_group_environment = is_new_group_environment
+        self.is_reappeared = is_reappeared
```

Comment:
The mix of present tense `is` and past tense `reappeared` is a bit awkward. Would `has_reappeared` work?

Reply:
Yep. Changed!